### PR TITLE
add support for POST/PUT call with headers

### PR DESCRIPTION
**Pull request by xsuoownit**

### CODE REVIEW

File: src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java [hunk 1]
```diff
@@ -99,6 +114,20 @@ public Response successfulPOST(final String body) {
         return Response.created(uriInfo.getAbsolutePathBuilder().path("1").build()).build();
     }
 
+    @PUT
+    @Path("/successful/headers/put")
+    @Consumes("application/json")
+    public Response successfulHeadersPUT(final String body) {
+        return Response.noContent().build();
```
Comment (cchacin) [inline src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java]: Can you PUT the request headers in the response and check it in the feature file?

Just to make sure the we are sending the headers properly

File: src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java [hunk 2]
```diff
@@ -99,6 +114,20 @@ public Response successfulPOST(final String body) {
...
+    }
+
+    @POST
+    @Path("/successful/headers/post")
+    @Consumes("application/json")
+    public Response successfulHeadersPOST(final String body) {
+        return Response.created(uriInfo.getAbsolutePathBuilder().path("1").build()).build();
```
Comment (cchacin) [inline src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java]: Same here

File: src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java (hunk 1 above)
Comment (xsuoownit) [inline src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java]: fix https://github.com/xsuoownit/cucumber-common-steps/commit/7fefdd3fe9d8d622886fbccce5ae42fa043a8e42

File: src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java (hunk 2 above)
Comment (xsuoownit) [inline src/test/java/com/github/cchacin/cucumber/steps/example/app/Controller.java]: fix https://github.com/xsuoownit/cucumber-common-steps/commit/7fefdd3fe9d8d622886fbccce5ae42fa043a8e42

**Comment by cchacin** [comment]
LGTM :+1:

**Pull request merged**